frontend/src/types/schemaEditor/index.ts

```diff
@@ -96,6 +101,16 @@ export interface AddColumnContext {
   default?: string;
 }
 
+export interface AlterColumnContext {
+  oldName: string;
+  newName: string;
+  type?: string;
+  comment?: string;
+  nullable?: boolean;
+  defaultChanged: boolean;
+  default?: string;
+}
+
```

Comment:
Special reasons that we need a new type other than `ChangeColumnContext`?

Reply:
`ChangeColumnContext` is using for MySQL to change column which needs the full structure information. 

`AlterColumnContext` has only those changed fields and using for PG with our parser.